client/src/pages/hostSetup.tsx

```diff
@@ -0,0 +1,24 @@
+import styles from "../styles/hostSetup.module.css";
+import Textbox from "../components/Textbox";
+import Checkbox from "../components/Checkbox";
+import Button from "../components/Button";
+const hostSetup = () => {
+    return(
+        <div className={styles.main}>
+            <div className={styles.title}>Host a Room</div>
+            <div className={styles.grid}>
+                <Textbox placeholder="Room Name" />
+                <Checkbox text="Only show games that everyone has" />
+                <div className={styles.enableAuth}>
+                    <div className={styles.padding}>
+                    <Checkbox text="Enable Authentication" />
+                    </div>
+                    <Textbox placeholder="Password"/>
```

Comment:
Should be using `<input type="password">`:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/password

Reply:
I have updated this but linked to the existing Textbox styling! 🧑‍💻